01-atoms/hr/Hr.component.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Hr = () => {
+  return <hr class="divider" />;
+};
+
+export default Hr;
```

Comment:
Just for clarity, you can get rid of that return statement:
```
const Hr = () => <hr class="divider" />;

export default Hr;
```
Or if you really want to be brief you can export from the same line. I will typically do this for all components unless I have a reason not to.
```
export default const Hr = () => <hr class="divider" />;
```